setup/options.rst

```diff
@@ -0,0 +1,228 @@
+
+* **Use vertical sync**
+
+  When enabled, attempts to synchronize frames with the monitor's refresh rate, preventing screen tearing.
+
+* **Multithreading**
+
+  Experimental option which, when enabled, uses multiple threads to render.
+
+* **Minimise fullscreen on focus loss**
+
+  When enabled, if the mouse cursor leaves the window in fullscreen mode, minimize the game to the taskbar.
+
+
+Rendering
+---------
+
+* **Landscape Smoothing**
+
+  Enables/disables landscape smoothing on grid edges.
+
+  .. image:: _static/landscape_smoothing.gif
+
+* **Gridlines on Landscape**
+
+  Enables/disables gridlines on the landscape.
+
+  .. image:: _static/landscape_gridlines.png
+
+* **Display text on banners in uppercase**
+
+  If enabled, displays the text on banners (ride entrances, signs) in uppercase.
+
+  .. figure:: _static/banner_text_uppercase.png
+     :scale: 200 %
+
+     Uppercase text
+
+  .. figure:: _static/banner_text_lowercase.png
+     :scale: 200 %
+
+     Upper and lowercase text
+
+* **Show guest purchases as animation**
+
+  If enabled, shows when guests make a purchase as floating green text.
+
+  .. image:: _static/show_guest_purchases.png
+
+* **Transparent background for giant screenshots**
+
+  If enabled, giant screenshots will have a transparent background where the outside of the park would normally be, instead of the dark void.
+
+* **Virtual floor style**
+
+  Can be one of three values:
+
+  * *Disabled* - No virtual floor is shown when building scenery off ground level.
+  * *Clear (Transparent)* - An outline of the current building grid is shown beneath the scenery object being placed.
+  * *Glassy (Translucent)* - An outline of the current building grid is shown beneath the scenery object being placed, and the interior of the grid is a translucent olor.
+
+  .. image:: _static/virtual_floor_disabled.png
+     :scale: 70 %
+
+  .. image:: _static/virtual_floor_clear.png
+     :scale: 70 %
+
+  .. image:: _static/virtual_floor_glassy.png
+     :scale: 70 %
+
+* **Cycle day / night**
+
+  If enabled, the game will cycle between day and night cycles, causing the colour palette to change.
+
+  .. image:: _static/day_night_cycle.gif
+
+  * **Enable lighting effects**
+
+    If enabled, light-producing items such as lamps will emit a glow during nighttime and rainstorms.
+
+    * **Enable lighting effects on rides**
+
+      If enabled, some rides will emit a glow during nighttime and rainstorms
+
+* **Render weather effects**
+
+  If enabled, rain and gloomy colours will be rendered during storms
+
+  * **Disable lightning effect**
+
+    If enabled, the lightning effect during thunderstorms will not occur
+
+Culture and Units
+-----------------
+
+* **Language**
+
+  Used to select the language OpenRCT2 uses.
+
+* **Currency**
+
+  Used to select the currency OpenRCT2 uses.
+
+  Selecting "Custom currency" will bring up a menu allowing you to specify a currency not built into OpenRCT2.
```

Comment:
Use curly quotes: instead of straight ones 

```suggestion
  Selecting “Custom currency” will bring up a menu allowing you to specify a currency not built into OpenRCT2.
```

Reply:
Any particular reason for this one? Seems kind of arbitrary to me.

I made the change either way though.

Reply:
The curly quotes are usually considered the “proper” ones and recommended by most style guides. The straight quotes were introduced by typewriter manufacturers to save a key, which later carried over to ASCII. I don't mind straight quotes in a technical wiki article, but I prefer adhering more strictly to correct typography for this manual.